bbs/conf.cpp

```diff
@@ -52,7 +52,7 @@ static int disable_conf_cnt = 0;
 
 // Locals
 char* GetGenderAllowed(int nGender, char* pszGenderAllowed);
-int modify_conf(ConferenceType conftype, int which);
```

Comment:
You can delete this whole line, we don't need it declared

Reply:
why is changing it to void not the right thing to do?  like:
```c
void insert_conf(ConferenceType conftype, int n);
void delete_conf(ConferenceType conftype, int n);
```
below it?  I thought you needed function prototypes for everything?

Reply:
Nope, just if you call it before you implement it.